File: python/src/commands/CreateGreetingCommand.py
from typing import List
from src.commands import ICommand
from src.services import GreetingService


class CreateGreetingCommand(ICommand):
    def __init__(self, greetingService: GreetingService) -> None:
        self._greetingService = greetingService

    def invoke(self, tokens: List[str]) -> None:
        message = tokens[1]
        g = self._greetingService.create(message)
        print(g)

File: java/src/main/java/com/example/demo/commands/CreateGreetingCommand.java
package com.example.demo.commands;

import java.util.List;

import com.example.demo.entities.Greeting;
import com.example.demo.services.GreetingService;

public class CreateGreetingCommand implements ICommand{

    private final GreetingService greetingService; 

    public CreateGreetingCommand(GreetingService greetingService) {
        this.greetingService = greetingService;
    }

    @Override
    public void invoke(List<String> tokens) {
        String message = tokens.get(1);
        Greeting createGreet = greetingService.create(message);
        System.out.println(createGreet);
    }
    
}
